.github/workflows/docker-build.yml

```diff
@@ -48,15 +45,14 @@ jobs:
             # Use short SHA
             type=sha,format=short
             # Only use latest tag on actual releases
-            type=raw,value=latest
+            type=raw,value=latest,enable=${{ github.event_name == 'release' }}
 
       - name: Build Docker image
         uses: docker/build-push-action@v6
         id: push
         with:
           context: .
-          # TODO: Re-enable multi-platform support when Dockerfile supports arm64
-          # platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64
```

Comment:
Nice to have the [multi-platform builds](https://docs.docker.com/build/building/multi-platform/) so it's not limited to just x86.